### PR TITLE
2.0/develop

### PR DESCRIPTION
**Pull request by yschaeff**

### CODE REVIEW

**Comment by yschaeff** [comment]
#build

File: enforcer/src/ods-migrate.c [hunk 1]
```diff
@@ -220,11 +220,13 @@ dblayer_mysql_foreach(const char* listQueryStr, const char* updateQueryStr, int
     mysql_stmt_prepare(updateStmt, updateQueryStr, strlen(updateQueryStr) + 1);
     mysql_query(dblayer_mysql.handle, listQueryStr);
     res = mysql_store_result(dblayer_mysql.handle);
+    if (!res) {
+        fprintf(stderr, "Failed to update db. Is it set correctly in conf.xml?");
```
Comment (hodarh) [inline enforcer/src/ods-migrate.c]: Please add "\n" at the end of message

File: enforcer/src/ods-migrate.c [hunk 2]
```diff
@@ -246,10 +247,12 @@ static void
 dblayer_mysql_open(const char* host, const char* user, const char* pass,
         const char *rsrc, unsigned int port, const char *unix_socket)
 {
-    MYSQL* database;
     dblayer_mysql.handle = mysql_init(NULL);
-    database = mysql_real_connect(dblayer_mysql.handle, host, user, pass, rsrc, port, NULL, 0);
-    (void)database;
+    if (!mysql_real_connect(dblayer_mysql.handle, host, user, pass, rsrc, port, NULL, 0)) {
+	fprintf(stderr, "Failed to connect to database: Error: %s\n",
+	    mysql_error(&mysql));
```
Comment (hodarh) [inline enforcer/src/ods-migrate.c]: mysql is not defined

**Comment by yschaeff** [comment]
#build

**Comment by yschaeff** [comment]
#build

**Pull request merged**